tests/ui/README.md

```diff
@@ -15,20 +15,17 @@ it would be a good idea to create a fork of this repository first, and then clon
 GitHub also has great instructions for [forking a repository][git-fork].
 
 ### Run the tests
-Tests are run using tox on the command line. Below are a couple of examples of running the tests:
+The tests must be run in Firefox 48 or later.
 
-Before running, ensure the [GeckoDriver executable][geckodriver] (rename to wires) is in your path.
-
-Note: The tests must be run in Firefox 48 or later, and geckodriver (wires) must be 0.9.0.
-
-```bash
-$ tox -e ui-tests
-```
+1. Install [Tox]
+1. Download geckodriver [v0.10.0][geckodriver] and ensure it's executable and
+   in your path
+1. Run `tox -e discopane-ui-tests`
```

Comment:
I think this can just be `tox` since the envlist is pretty small in `.tox`. This should future proof the docs a bit.